visualSHARK/views.py

```diff
@@ -677,6 +720,8 @@ def get(self, request):
 class StatsView(APIView):
     # TODO: update to serializer
 
+    read_perm = 'view_analytics'
```

Comment:
view_stats, I would separate this from analytics, stats also encompasses the dashboard.